images/diagnostics/scripts/openshift-network-debug

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+#
+# This script is used by network diagnostics.
+# Based on containerized/non-containerized openshift install,
+# it sets the environment so that docker, openshift, iptables, etc.
+# binaries are availble for network diagnostics.
+#
+set -o nounset
+set -o pipefail
+
+node_rootfs=$1
+shift
+cmd="${@}"
+node_image="openshift/node"
+
+node_container_id=$(chroot "${node_rootfs}" docker ps | grep "${node_image}:" | cut -d' ' -f1)
```

Comment:
`node_container_id=$(chroot "${node_rootfs}" docker ps --filter="name=${node_image}" --format="{{.ID}}")` ?

likewise with the "docker inspect" below

also, guidelines say to put quotes around $()